.travis.yml

```diff
@@ -12,6 +12,7 @@ cache:
   directories:
     - travis_phantomjs
 before_install:
+  - npm config set registry http://ci.strongloop.com:4873/
```

Comment:
Same issue of cross-dependency ..
A different pr will not pass, as it needs  the content of the first commit .. The first commit will not pass without the 2nd of a juggler release ...
I didn't intend to squash anyways .. 2 different commits for 2 different purposes!